docker/oscar/bin/startup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euxo
+
+catalina.sh start
+
+rm -rf /usr/local/tomcat/webapps/ROOT
```

Comment:
I place a redirect in here as index.jsp , instead maybe we could keep that instead of deleting the ROOT app?

Also, can we remove all these in the Dockerfile? If it works that's preferable for simpler/quicker startups

Reply:
I tried removing through the Dockerfile but I wasn't able to. I'm guessing it's because when running tomcat the first time `catalina.sh run` that's when all the default apps are built.

Reply:
Ok, let's not delete ROOT, and use the (unfortunately also needed) volume I added with the redirect instead. You can delete the other apps.

Reply:
That ROOT directory contains all kinds of garbage. It's not needed.
My 2 cents regarding "unfortunately also needed"; ultimately it would be best to avoid putting any redirect in the ROOT directory and to use a proper Tomcat context redirect instead.  

Official documentation: https://tomcat.apache.org/tomcat-8.5-doc/config/context.html
Helpful tutorial: https://octopus.com/blog/defining-tomcat-context-paths
Another short and sweet tutorial: https://stackoverflow.com/questions/7276989/how-to-set-the-context-path-of-a-web-application-in-tomcat-7-0

@countable Clark; it's your call.

Reply:
The same (above) should be done with Oscar

Reply:
Sure. I see a lot of misalignment of the responses in that SO thread, gotchas and ways to do the same thing in tomcat. @warrendennis. Let's merge this and then add another ticket to replace the redirect. I also say this because running rm -fr on startup isn't exactly good either, so if we really want a clean setup we need to do more work from the ground up anyway. This at least works, and should do no harm I think.